src/submission/readFile.ts

```diff
@@ -82,8 +86,10 @@ export const parseFileToRecords = async (
 
 		stream.on('data', (record: string[]) => {
 			if (!headers.length) {
-				headers = record.map((value) => schemaDisplayNames[value] || value);
-				console.log(`header:${JSON.stringify(headers)}`);
+				headers = record
+					.map((value) => schemaDisplayNames[value] || value)
```

Comment:
I know types are a thing, but runtime is funny, and this handles null values ;)
```suggestion
					.map((value) => schemaDisplayNames[value] ?? value)
```